### PR TITLE
Voice command confirmation

### PR DESCRIPTION
**Pull request by nk-io**

### CODE REVIEW

File: client/app/src/main/java/group01/smartcar/client/activity/DrivingActivity.java [hunk 1]
```diff
@@ -484,12 +484,40 @@ private void onSpeechResults(Bundle bundle) {
         System.out.println(data.get(0));
         final String[] commandParts = command.trim().split(" ");
 
+        final boolean executionResult = executeVoiceCommand(commandParts);
+
+        // TODO: Give visual feedback for command execution result
+        if(executionResult == true){
```
Comment (JensTheS) [inline client/app/src/main/java/group01/smartcar/client/activity/DrivingActivity.java]: This can be simplified to if(executionResult)

**Review by JensTheS — CHANGES_REQUESTED** [review]
All in all looks good, suggest one change.

File: client/app/src/main/java/group01/smartcar/client/activity/DrivingActivity.java (hunk 1 above)
Comment (nk-io) [reply client/app/src/main/java/group01/smartcar/client/activity/DrivingActivity.java]: updated!

**Review by JensTheS — APPROVED** [review]
Nice work!

**Pull request merged**